File: Microverse Challenges/hackerrank/fizzbuzz.js
function fizzbuzz(num) {
  for (var i = 1; i<=num; i++) {
    var output = '';
    if (i%3 == 0) { output += 'Fizz'; }
    if (i%5 == 0) { output += 'Buzz'; }
    if (output == '') { output = i; }
    console.log(output);
  }
}

console.log(fizzbuzz(100));

File: Microverse Challenges/hackerrank/fizzbuzz.rb
(1..100).each do |i|
  puts i unless (i % 15).zero? || (i % 3).zero? || (i % 5).zero?
  if (i % 15).zero?
    puts 'FizzBuzz'
  elsif (i % 3).zero?
    puts 'Fizz'
  elsif (i % 5).zero?
    puts 'Buzz'
  end
end
